dissect/target/loaders/libvirt.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+
+from defusedxml import ElementTree
+
+import dissect.target.container as container
+from dissect.target import Target
+from dissect.target.loader import Loader
+
+
+class LibvirtLoader(Loader):
+    """Load libvirt xml configuration files."""
+
+    def __init__(self, path: Path, **kwargs):
+        path = path.resolve()
+
+        super().__init__(path)
+
+    @staticmethod
+    def detect(path: Path) -> bool:
+        if path.suffix.lower() != ".xml":
+            return False
+
+        with path.open("rb") as fh:
+            return b"<domain>" in fh.read(512)
```

Comment:
This doesn't seem to match the `qemu.xml` that you included in the test file. Also `<domain>` seems a little too generic, can you make this check more strict?